whois-api/src/test/java/net/ripe/db/whois/api/sso/AuthServiceClientTestIntegration.java

```diff
@@ -61,4 +85,54 @@ public void get_historical_event_date_time() {
         assertThat(historicalUserDetails.response.results.get(0).eventDateTime, is(LocalDateTime.of(2015, 5, 8, 12, 32, 0)));
     }
 
+
+    // Errors
+    @Test
+    public void get_validate_token_response_null_input_then_error() {
+        assertThat(cacheManager.getCache("ssoValidateToken").get(UUID), is(nullValue()));
+
+        final AuthServiceClientException authServiceClientException = assertThrows(AuthServiceClientException.class, () -> {
+            authServiceClient.validateToken(null);
+        });
+
+        assertThat(authServiceClientException.getMessage(), is("No UUID."));
```

Comment:
Should be no token instead of no UUID?

Reply:
I did not change the returned message of the other class (AuthService)
Here UUID and Token are the same. I can change it, but they are the same

Reply:
UUID and token are not same. UUID is assigned against an email id and Token is used for authentication purposes. Correct me if I am wrong. Can you change it ?

Reply:
You are right, the token is in the header as a bearer token and it is used for auth.
The UUID is the SSO identification for the user

I mean in tests code is the same, we are using same value for both (same value for variable UUID). I will refactor this, to use a different value, the test will be clear and return token instead uuid

Reply:
Updated, now the Token looks like a token in the tests and not like an UUID